server/app/lib/crm.js

```diff
@@ -0,0 +1,75 @@
+const {ov_config} = require("./ov_config")
+const axios = require("axios")
+
+async function getAmbassadorHSID(email) {
+  let hs_key = ov_config.hubspot_api_key
+
+  console.log(`[HS] Calling API with: ${email}`)
+
+  try {
+    let response
+    if (hs_key) {
+      response = await axios.post(
+        `https://api.hubapi.com/crm/v3/objects/contacts/search?hapikey=${hs_key}`,
+        {
+          filterGroups: [
+            {
+              filters: [
+                {
+                  propertyName: "email",
+                  value: email,
+                  operator: "EQ",
+                },
+              ],
+            },
+          ],
+        },
+      )
+    }
+    updateHubspotAmbassador(response.data.results[0].id)
```

Comment:
it looks like the updateHubspotAmbassador function below is expecting a req object, but here it looks like we're passing it an id? but won't that just be a string? I might be misunderstanding something.

Reply:
i think i left this here for testing by accident

Reply:
removed